docs/dm/zh_CN/features/table-route.md

```diff
@@ -46,12 +60,37 @@ routes:
     target-schema: "test"
     
 ```
+***
+
+#### 分库合并
+
+假设存在分库场景 - 将上游两个 MySQL 实例 `test_{1,2,3...}`.`t_{1,2,3...}` 同步到下游 TiDB 的 `test`.`t_{1,2,3...}`
 
-- 同步到下游的 `test`.`t_[1,2,3]`，则必须创建上面参数配置的一条路由规则
+- 同步到下游的 `test`.`t_{1,2,3...}`，创建一个路由规则即可
+```yaml
+  rule-1:
+    schema-pattern: "test_*"
+    target-schema: "test"
 ```
+
+***
+
+#### 错误的路由
+
+假设存在下面两个路由规则，`test_1_bak`.`t_1_bak` 可以匹配上 `rule-1` 和 `rule-2`, 违反 table 路由的限制而报错
```

Comment:
declare will report an error until syncing `test_1_bak`.`t_1_bak`, not starting the task.